contents/data/sql/SQL.md

```diff
@@ -0,0 +1,145 @@
+
+### Highly Compatible
+
+SQL software works on many operating systems (%%like Windows, Mac OS, Ubuntu, Red Hat among others%%) and more importantly, can be integrated with many languages. This includes programming languages like C++, Ruby, Java, Python, JavaScript (Node.js), and Go.
+
+In addition, online playgrounds like [DB Fiddle](https://www.db-fiddle.com/) and [JDoodle](https://www.jdoodle.com/execute-sql-online) make it easy to use and learn the SQL language quickly without the hassle of setting up an environment or application.
+
+## Disadvantages
+
+SQL is not without its problems. In general, the biggest problem is with regards to the features of SQL.
+
+Although SQL databases use established <tooltip content="American National Standard Institutes">ANSI</tooltip> & <tooltip content="International Organization for Standardization">ISO</tooltip> standards, some SQL software systems (%%PostgreSQL, for example%%) add proprietary extensions to standard SQL.
+Due to this, the available feature set can vary according to what you're using. This can make SQL confusing and frustrating to use when switching SQL software.
+
+Apart from that, most **SQL** problems are not uniformly applicable across all SQL software. For example, MySQL suffers from concurrency issues. Though it performs well with read operations, it *can* be problematic when there are many concurrent read-write operations. A symptom of this issue would be a sudden slowdown of a query. However, PostgreSQL deals with concurrency well by implementing <tooltip content="each query transaction is isolated from other transactions running simultaneously in the system">transactions isolation</tooltip>.
+
+---
+
+## How to get started with SQL?
+
+The first thing to do when getting started with SQL is to choose a SQL software. You can either just use an online SQL playground or install a dedicated application. Our suggestion is to use either MySQL or SQLite online.
+
+SQL has been around for a long time and hence, you can find really good books and resources to learn it well. However, these resources can be overwhelming. So, you can decide which parts you want to learn, as you will likely not need to know everything to get started with using a SQL database.
+
+Here are some recommended steps for learning SQL:
+
+- Understand what a database and [<tooltip content="DataBase Management System">DBMS</tooltip>]({{baseUrl}}/contents/data/databases/databases.html) are. More specifically, understand the basic concepts of [RDBMS](https://www.tutorialspoint.com/sql/sql-rdbms-concepts.htm). This is not _essential_, but will give you a better high-level understanding before diving into programming.
```

Comment:
To be more useful, you should point to where the reader can go to accomplish this step.